python/test_pip_package.py

```diff
@@ -515,5 +516,149 @@ def testVersion(self):
         tf.compat.as_bytes(stdout))
 
 
+class ConvertTfKerasSavedModelTest(tf.test.TestCase):
+
+  def setUp(self):
+    super(ConvertTfKerasSavedModelTest, self).setUp()
+    self._tmp_dir = tempfile.mkdtemp()
+
+  def tearDown(self):
+    if os.path.isdir(self._tmp_dir):
+      shutil.rmtree(self._tmp_dir)
+    super(ConvertTfKerasSavedModelTest, self).tearDown()
+
+  def _createSimpleSequentialModel(self):
+    model = tf.keras.Sequential()
+    model.add(tf.keras.layers.Reshape([2, 3], input_shape=[6]))
+    model.add(tf.keras.layers.LSTM(10))
+    model.add(tf.keras.layers.Dense(1, activation='sigmoid'))
+    return model
+
+  def _createNestedSequentialModel(self):
+    model = tf.keras.Sequential()
+    model.add(tf.keras.layers.Dense(6, input_shape=[10], activation='relu'))
+    model.add(self._createSimpleSequentialModel())
+    return model
+
+  def _createFunctionalModelWithWeights(self):
```

Comment:
Can you add a test including a compiled model with a few training steps? Note that only tf.optimizers work with the Keras export.